xbmc/cores/VideoPlayer/Process/VideoBuffer.cpp

```diff
@@ -318,14 +318,21 @@ void CVideoBufferPoolSysMem::Return(int id)
   m_free.push_back(id);
 }
 
-void CVideoBufferPoolSysMem::Configure(AVPixelFormat format, int width, int height)
+void CVideoBufferPoolSysMem::Configure(AVPixelFormat format, int width, int height, int aligned_width, int aligned_height, int size)
```

Comment:
aligned width / height are used in the Pi implementation, as that information gets passed back to renderer.
But lets see if I can just use the size on Configure, and work out the aligned_width/aligned_height from GetStride/GetPlanes in the renderer.